doc/source/ray-core/actors/async_api.rst

```diff
@@ -67,6 +67,7 @@ async frameworks like aiohttp, aioredis, etc.
     # NOTE: The outputs from the previous code block can show up in subsequent tests.
     # To prevent flakiness, we wait for the async calls finish.
     import time
+    print("Sleeping...")
```

Comment:
what is this for?

Reply:
Usually, there are outputs from the previous code block (which asynchronously prints out stuff), and this code block is intended to catch them. However, sometimes everything actually gets caught in the previous `testoutput` block. In this case, there are no outputs from this code block, and the test errors. 

This print ensures there are always some outputs from this code block.